packages/components/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -1,8 +1,7 @@
 import React from "react";
-import { styled, theme } from "@localyze-pluto/theme";
```

Comment:
By importing theme.colors from this package it would never be able to get the tokens that are overwritten on the ThemeProvider